pkg/cmd/run/download/zip.go

```diff
@@ -42,14 +49,14 @@ func extractZipFile(zf *zip.File, dest string) (extractErr error) {
 	}
 	defer f.Close()
 
-	if dir := filepath.Dir(dest); dir != "." {
+	if dir := filepath.Dir(dest.String()); dir != "." {
 		if extractErr = os.MkdirAll(dir, dirMode); extractErr != nil {
 			return
 		}
 	}
```

Comment:
With `dest` becoming an absolute path, is there any possibility it would be `.`?